DOCUMENTATION.md

```diff
@@ -49,66 +49,66 @@ client.close()
+Add a callback function, to be called when an uplink message is received.
 ```python
 client.set_uplink_callback(uplink_callback)
 ```
 
 ##### uplink_callback
 The callback function must be declared in the script following this structure:
 * `uplink_callback(msg, client)`
-  * `msg`: **JSON**  the message received by the client
-  * `client`: **object**  the client from which the callback is executed are calling
+  * `msg`: **object**  the message received by the client.
+  * `client`: **object**  the client from which the callback is executed.
 
 #### set_connect_callback
-Set a connection callback function to be executed when the client connects to the broker. It's possible to add more than one callback function.
+Add a connection callback function to be executed when the client connects to the broker.
 ```python
 client.set_connect_callback(connect_callback)
 ```
 ##### connect_callback
 - `connect_callback(res, client)`: the function which will be executed on connection to the broker.
-  - `res`: **boolean**  the result of the connection. If it's true, the connection succeeded went well. If not, it means the connection failed.
+  - `res`: **boolean**  the result of the connection. If it's true, the connection succeeded. If not, it means the connection failed.
   - `client`: **object**  the TTN client from which the callback is called.
 
 #### set_downlink_callback
-Set a downlink callback function, with actions to execute when a downlink message is sent. It's possible to add more than one callback function.
+Add a downlink callback function, with actions to execute when a downlink message is sent.
 ```python
 client.set_downlink_callback(downlinkCallback)
 ```
 ##### downlink_callback
-- `downlink_callback(mid, client)`: the function which will be the new publish behavior for our MQTT client.
+- `downlink_callback(mid, client)`: the function which will be a new publish behavior for our MQTT client.
   - `mid`: **int**  this is the message ID for the downlink request. It can be used to track the request.
   - `client`: **object**  the TTN client from which the callback is called.
 
 #### set_close_callback
-Set a callback to be executed when the connection to the TTN broker is closed. It's possible to add more than one callback function.
+Add a callback to be executed when the connection to the TTN broker is closed.
 ```python
 client.set_close_callback(close_callback)
 ```
 ##### close_callback
 - `close_callback(res, client)`: the function which will be executed when the connection is closed.
-  - `res`: **boolean**  the result of the disconnection. If it's true, it went well. If not, it means the disconnection was unexpected.
+  - `res`: **boolean**  the result of the disconnection. If it's true, it went all as expected. If not, it means the disconnection was unexpected.
   - `client`: **object**  the TTN client from which we call the callback.
```

Comment:
Please clarify whether it is an `Object` (and in that class, the class of this object) or if it is a `dict`